etc/config-template.toml

```diff
@@ -61,6 +61,11 @@
 # Amount to read ahead on individual grpc streams.
 # grpc-stream-initial-window-size = "2MB"
 
+# How many snapshots can be sent concurrently.
+# concurrent-send-snap-limit = 32
```

Comment:
Is the default value too high?

Reply:
No. For most server's configuration, 32 is OK.